### DISCUSSION TITLE
Finetuning Voice Conversion

### DISCUSSION THREAD

**Discussion by neuronx1** [coqui-ai/TTS#1314] [General Q&A]
Hello,
thanks for your great repository.

I have one question:

Is it possible to finetune a voice conversion model instead doing zero-shot (https://colab.research.google.com/drive/1gjdwOKCZuavPn_5oy8QA01sKmXpEq5AZ?usp=sharing#scrollTo=uSgpKI-ZcVbx).
So actually I want to finetune a voice conversion model on one source and one target speaker to improve the output quality.

If yes, do you have some finetuning recipes for this problem?

Thanks in advance!

**Comment by TejaswiniiB** [comment]
Hi @neuronx1 
Think Speaker Adaptation mentioned in the [blog ](https://coqui.ai/blog/tts/yourtts-zero-shot-text-synthesis-low-resource-languages) of this repo does similar to what you have asked.
There seems to be a way to finetune a pre-trained model with this project.
Details reg. it are provided in their documentation [here](https://tts.readthedocs.io/en/latest/finetuning.html).

**Comment by Edresson** [comment]
> Hello, thanks for your great repository.
> 
> I have one question:
> 
> Is it possible to finetune a voice conversion model instead doing zero-shot (https://colab.research.google.com/drive/1gjdwOKCZuavPn_5oy8QA01sKmXpEq5AZ?usp=sharing#scrollTo=uSgpKI-ZcVbx). So actually I want to finetune a voice conversion model on one source and one target speaker to improve the output quality.
> 
> If yes, do you have some finetuning recipes for this problem?
> 
> Thanks in advance!

The fine-tuning for voice conversion is equal to the fine-tuning for the TTS. So the speaker adaptation for TTS and Voice conversion follows the same process.

We currently do not provide guidelines for fine-tuning YourTTS due to the potential for misuse of this model. We intend to make these guidelines available in the future as soon as we have an efficient way to detect fake audio/generated speech. This topic is also discussed in #1196 and [here](https://github.com/Edresson/YourTTS/issues/5).
